ovirtlib4/data_centers.py

```diff
@@ -27,3 +27,31 @@ class DadaCenterEntity(CollectionEntity):
     """
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+    @property
+    def storage_domains(self):
+        return DataCenterStorageDomains(connection=self.service)
+
+
+class DataCenterStorageDomains(CollectionService):
+    """
+    Gives access to all VM NICs
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.service = self.connection.storage_domains_service()
+        self.entity_service = self.service.storage_domain_service
+        self.entity_type = types.StorageDomain
+
+    def _get_collection_entity(self):
+        """ Overwrite abstract parent method """
+        return DataCenterStorageDomain(connection=self.connection)
+
+
+class DataCenterStorageDomain(CollectionEntity):
+    """
+    Put StorageDomain custom functions here
```

Comment:
I would replace 'Put' with something more explanatory.
Using a directive in a class doc-string does not feel right to me

Reply:
Currently, all doc-string are using this word, so I prefer to keep the convention.